pkg/api/router.go

```diff
@@ -81,6 +81,15 @@ func (s *server) setupRouting() {
 		),
 	})
 
+	handle(router, "/chunks-pin/{address}", jsonhttp.MethodHandler{
```

Comment:
maybe just `/pin`?

Reply:
We should have also endpoints to pin bytes, files and collections. I also do not like so much chunks-pin. Maybe to have:

```
/pinning/chunks
/pinning/bytes
/pinning/files
/pinning/dirs
```

For now just `/pinning/chunks`?

Reply:
SGTM :+1:

Reply:
Cool, thanks. Changed.